src/main/java/ch/njol/unofficialmonumentamod/UnofficialMonumentaModClient.java

```diff
@@ -51,27 +78,68 @@ public void onInitializeClient() {
 			e.printStackTrace();
 		}
 
+		if (options.discordEnabled) discordPresence.init();
+
 		ClientTickEvents.END_CLIENT_TICK.register(client -> {
 			abilityHandler.tick();
+			KeybindingHandler.tick();
+			Calculator.tick();
+			CooldownManager.update();
+		});
+
+		ClientTickEvents.END_WORLD_TICK.register(world -> {
+			Notifier.tick();
 		});
 
+		ClientCommandManager.DISPATCHER.register((ClientCommandManager.literal("spoof").then(ClientCommandManager.argument("name", StringArgumentType.greedyString()).executes(ItemNameSpoofer::commandNameSpoofer))));

```

Comment:
not really just cosmetic

Reply:
the only thing I can say about that is that it can look cool and I remember using a similar feature on another server to know when I lent my items if they were actually mine.

Reply:
Since Monumenta items are not distinguishable from one another you won't be able to accomplish this. The best way is to infuse or stat track an item to mark it with your name, which doesn't require a client mod. I'm very hesitant to add this feature, as it's hardly useful and I'll probably fall onto me to keep supporting it.

Reply:
It seems I didn't think that through